lib/ipmi_fru.c

```diff
@@ -4855,30 +4809,23 @@ f_type, uint8_t f_index, char *f_string)
 				header_offset, fru_section_len, fru_data) < 0 )
 		{
 			printf("Write to FRU data failed.\n");
-			rc = (-1);
+			rc = -1;
 			goto ipmi_fru_set_field_string_out;
 		}
 	}
 	else {
 		printf("String size are not equal, resizing fru to fit new string\n");
-		if(
-				ipmi_fru_set_field_string_rebuild(intf,fruId,fru,header,f_type,f_index,f_string)
-		)
-		{
-			rc = (-1);
+		if (ipmi_fru_set_field_string_rebuild(intf, fruId, fru, header,
+						      f_type, f_index,
```

Comment:
Bad indentation again.

Reply:
This change is also not related to the commit and will be dropped.